
def error(msg, exit_code = 1):
    print(f"taskmaster: {msg}")
    exit(exit_code)

